wemake_python_styleguide/visitors/ast/classes.py

```diff
@@ -121,27 +120,23 @@ def _is_correct_base_class(self, base_class: ast.AST) -> bool:
         return False
 
     def _check_getters_setters_methods(self, node: ast.ClassDef) -> None:
-        class_attributes, instance_attributes = classes.get_all_attributes_str(
-            node,
-        )
-        class_postfixes, instance_postfixes = classes.get_set_postfixes(node)
-
-        if variables.name_in_postfix(node, class_attributes, class_postfixes):
-            self.add_violation(
-                oop.UnpythonicGetterSetterViolation(
-                    node,
-                    text=node.name,
-                ),
-            )
-
-        all_attributes = instance_attributes.union(class_attributes)
-        if variables.name_in_postfix(node, all_attributes, instance_postfixes):
-            self.add_violation(
-                oop.UnpythonicGetterSetterViolation(
-                    node,
-                    text=node.name,
-                ),
-            )
+        class_attributes, instance_attributes = classes.get_all_attributes(node)
+        attribute_names = {
```

Comment:
I guess we can even more this logic into `get_all_attributes` function. Currently it does not have any logic inside at all.

Reply:
That function isnt needed, see my latest commit. I think it should still be plain as it is so that it can be reused later for whatever.